mysys/crc32/crc32_arm64.c

```diff
@@ -137,9 +137,9 @@ asm(".arch_extension crypto");
 #define CRC32B(crc, value) (crc) = __crc32b((crc), (value))
 
 #define CRC32C3X8(buffer, ITR) \
-  crc1 = __crc32cd(crc1, *((const uint64_t *)buffer + 42*1 + (ITR)));\
-  crc2 = __crc32cd(crc2, *((const uint64_t *)buffer + 42*2 + (ITR)));\
-  crc0 = __crc32cd(crc0, *((const uint64_t *)buffer + 42*0 + (ITR)));
+  crc1 = __crc32cd(crc1, uint8korr(buffer + 42*1 + (ITR)));\
+  crc2 = __crc32cd(crc2, uint8korr(buffer + 42*2 + (ITR)));\
+  crc0 = __crc32cd(crc0, uint8korr(buffer + 42*0 + (ITR)));
```

Comment:
Shouldn't asm version of `CRC32C3X8` a few lines above be fixed as well?

Reply:
Right, I forgot to test/adjust the asm version. Noted.